telegram/utils/helpers.py

```diff
@@ -19,13 +19,26 @@
 """This module contains helper functions."""
 
 import re
+import signal
 from datetime import datetime
 
 try:
     from html import escape as escape_html  # noqa: F401
 except ImportError:
     from cgi import escape as escape_html  # noqa: F401
 
+
+# From https://stackoverflow.com/questions/2549939/get-signal-names-from-numbers-in-python
+_signames = {v: k
+             for k, v in reversed(sorted(vars(signal).items()))
```

Comment:
nitpicking here. the `reversed` & `sorted` are useless here as you're creating a dict. please remove them.

Reply:
I think the `sorted` was there for signals with different names but same number, so there's a deterministic outcome. However I haven't understood the `reversed` call too^^

Reply:
And kk `reversed` then picks signal names that are sorted at the beginning. On my system I get these difference when I don't use `reversed`:
```
SIGIOT vs SIGABRT
SIGCLD vs SIGCHLD
SIGPOLL vs SIGIO
```

Though not sure if it's that important here^^

Reply:
Check manpage of `signal(7)` all the above are synonyms. If one name is chosen over another it's merely a matter of luck, as dictionary order is not guaranteed.
On my system I had different results on different executions.

Another option is to leave the signal name alone and just print the number.

Reply:
> If one name is chosen over another it's merely a matter of luck, as dictionary order is not guaranteed.

Looks like it's true, I'm just wondering: the order we iterate over should always produce a guaranteed outcome. `vars(signal).items()` has unique mapping from `name -> number`. Using `sorted` we get always the same list with the same order, and then we are filling the dict with those values, while switching `name` and `number`. If there are numbers that are equal, later elements in the list should overwrite the old ones. Although using dict-comprehension, citing the `dict`-docs:
>  If a key occurs more than once, the last value for that key becomes the corresponding value in the new dictionary.

Reply:
> Another option is to leave the signal name alone and just print the number.

I wouldn't do that, signal numbers differ on OSes for the "same" signals.

Reply:
ok. not worth the argument, lets keep it as is.